ci/azure/linux.yml

```diff
@@ -216,6 +216,11 @@ jobs:
     LOADS: "impala postgres"
 
   steps:
+    - bash: |
+        python pip install git+https://github.com/datapythonista/pysuerga
```

Comment:
should release this on pypi at some point

Reply:
Agree, will also release in conda-forge. Once this is merged, I'll see if any change to the current version is needed to make the package work for pandas, and will publish in pypi and conda-forge the version that works for both Ibis and pandas. It may save me a release. :rofl: